.snyk

```diff
@@ -0,0 +1,11 @@
+# Snyk (https://snyk.io) policy file, patches or ignores known vulnerabilities.
+version: v1.25.0
+# ignores vulnerabilities until expiry date; change duration by modifying expiry date
+exclude:
+  global:
```

Comment:
It's not obvious to me why we need to exclude these sources. Please add comments explaining why with a few details about the root problem.

Also, we shouldn't need the wildcard pattern matching on the last two. I expect something like this should work (untested):
`**/fetch-uj-records/fetch-uj-records_test.go**` -> `fetch-uj-records/fetch-uj-records_test.go`

Reply:
I will leave a comment there. I excluded them because it threw an error for using hardcoded passwords for having filePathPass in those files. @amisstea

Reply:
For anyone else following, snyk disagrees with our choice of naming for the `filePathPass` variables in our test files. Simply renaming those variables resolved the errors.